.circleci/Dockerfile-Nightly

```diff
@@ -0,0 +1,7 @@
+FROM brandonpage/salesforce-mobilesdk-android-docker-primary:0.1.0
```

Comment:
Second docker image (based on the first) created for nightly builds to keep PR's as fast as possible.  It is very likely this file will add additional API versions in the future and the extra size adds several minutes to the setup *and* each libraries test run due to additional size.